src/pages/patientView/PatientViewPage.tsx

```diff
@@ -449,6 +449,7 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
                                                 columnVisibilityProps={{
                                                     onColumnToggled: this.onMutationTableColumnVisibilityToggled
                                                 }}
+                                                showAllExon={true}
```

Comment:
@leexgh i think showTotalNrOfExons might be better to understand as variable name